README/ReleaseNotes/v634/index.md

```diff
@@ -104,6 +105,11 @@ The following people have contributed to this new version:
 
 * Setting `useHashMapForFind(true)` is not supported for RooArgLists anymore, since hash-assisted finding by name hash can be ambiguous: a RooArgList is allowed to have different elements with the same name. If you want to do fast lookups by name, convert your RooArgList to a RooArgSet.
 
+* function `bindFunction` now supports arbitrary many inputs.
+
+* class `RooPyBindFunction` can be used to wrap aritrary python functions. 
```

Comment:
```suggestion
```
You should not mention this. This is an implementation detail that the user should not interact with themselves: they will only interact with `RooFit::bindFunction()`.